internal/nomad/nomad.go

```diff
@@ -376,7 +379,11 @@ func handlePendingAllocationEvent(alloc *nomadApi.Allocation,
 			start:             time.Now(),
 			allocNomadNode:    alloc.NodeName,
 		})
-	} else {
+	case structs.AllocDesiredStatusStop:
+		// We delete the local data of this allocation / runner as we will likely receive event re-creating it.
```

Comment:
```suggestion
		// We delete the local data of this allocation / runner as we will likely receive an event re-creating it. If not, the next restart of Poseidon will re-add it.
```

Reply:
Does this imply that we are also recreating a new runner, in case the pre-warming pool decreased below the desired value?

Reply:
No. As you stated, a decreased pre-warming pool is currently refilled only on the next restart of Poseidon.